README.rst

```diff
@@ -184,4 +184,6 @@ Most of the API is available:
     client.users.get_crosstable
     client.users.get_user_performance
 
+    client.explorer.get_lichess_games
+
```

Comment:
We decided to sort the clients alphabetically here.

Reply:
Ok, noted !